src/nimble_boundary_condition_manager.h

```diff
@@ -92,7 +92,7 @@ class BoundaryConditionManager
 
   template <typename ViewT>
   void
-  ApplyInitialConditions(const ViewT reference_coordinates, ViewT velocity, std::vector<ViewT>& offnom_velocities)
+  ApplyInitialConditions(const ViewT reference_coordinates, ViewT velocity)
```

Comment:
I believe the variable 'offnom_velocities' is used when the code is running a UQ simulation.

@rjones-sandia may confirm.

Reply:
Reverted commits that removed these variables.